website/content/docs/plugins/creation/custom-datasources.mdx

```diff
@@ -1,7 +1,6 @@
 ---
 description: >
-  Packer Data Sources are the components of Packer that allow data to be fetched for use within the Packer configuration.
-  Use of data sources allows a build to use information defined outside of Packer.
+  Data sources fetch data to use in the Packer configuration. Learn how to define custom data sources so that a Packer can use external data in builds.
```

Comment:
```suggestion
  Data sources fetch data to use in the Packer configuration. Learn how to define custom data sources so that Packer can use external data in builds.
```